docker-compose-prod.yml

```diff
@@ -20,18 +20,17 @@ services:
       - database
 
   database:
-    image: mysql:5.7
-    command: ['--character-set-server=utf8mb4', '--collation-server=utf8mb4_unicode_ci']
+    image: mysql:8-oracle
+    command: ['--character-set-server=utf8mb4', '--collation-server=utf8mb4_unicode_ci', '--socket=/tmp/mysql.sock']
     environment:
       - MYSQL_ROOT_PASSWORD=ccm_root_pw
       - MYSQL_HOST=ccm_db_prod
-      - MYSQL_PORT=3306
+      - MYSQL_TCP_PORT=3306
       - MYSQL_DATABASE=ccm
       - MYSQL_USER=admin
       - MYSQL_PASSWORD=admin
-    entrypoint: ['docker-entrypoint.sh', '--default-authentication-plugin=mysql_native_password']
     ports:
       - 5400:3306
     volumes:
-      - ./.mysql_data_prod:/var/lib/mysql:delegated
+      - ./.mysql_data_prod/mysql:/var/lib/mysql
```

Comment:
This isn't a huge deal but changing the mount point of `./.mysql_data_prod` to `./.mysql_data_prod/mysql` will remove the existing database. Though you might have had to remove it anyway to upgrade.

Reply:
Ok, I changed it. just to keep it as before.